tests/integration/devfile/cmd_devfile_catalog_test.go

```diff
@@ -91,7 +91,7 @@ var _ = Describe("odo devfile catalog command tests", func() {
 	Context("When executing catalog describe component with a component name with a single project", func() {
 		It("should only give information about one project", func() {
 			output := helper.CmdShouldPass("odo", "catalog", "describe", "component", "java-openliberty")
-			helper.MatchAllInOutput(output, []string{"location: https://github.com/odo-devfiles/openliberty-ex.git"})
+			helper.MatchAllInOutput(output, []string{"location: https://github.com/OpenLiberty/application-stack.git"})
```

Comment:
This should ideally match for someting that doesn't depend on content of the devfile. 
For for the following regexp `location: .+`
`Expect(output).To(MatchRegexp("location: .+"))`

Reply:
> Expect(output).To(MatchRegexp("location: .+"))

+1